pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 100
```

Comment:
The default value seems to be 88, it appears that it was totally overthought: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

I don't mind if we change it to `<= 100` (but maybe we don't want to change it if nobody has an opinion about it).

Reply:
Generally, you either see 80 or 100 so I'm surprised with your choice of 99. :)

Reply:
\* "their choice of 88" ?

> In general, 90-ish seems like the wise choice.

I agree that's a funny choice, their justification appears to be "88 is 10% more than 80" :D